lms/templates/course.html

```diff
@@ -5,6 +5,10 @@
 from django.core.urlresolvers import reverse
 %>
 <%page args="course" expression_filter="h"/>
+<%
+    container_string = _("Starts: {date}")
+    date = course.retrieve_datetime()
```

Comment:
Similarly, a top-level variable named `date` should be specifically renamed to something like `course_start_date`.